lib/Varien/Image.php

```diff
@@ -71,6 +80,18 @@ public function open()
     }
 
     /**
+     * Free resources
+     *
+     * @return void
+     */
+    public function close() {
```

Comment:
No reason to add this method. The contained code should live directly in __destruct() instead.